src/bundles.sh

```diff
@@ -0,0 +1,18 @@
+export CONDUCTR_IP=$(hostname -i)
+
+conduct load conductr-haproxy
+conduct run conductr-haproxy --scale $NODE_COUNT
+
+sudo service haproxy restart
+
+sleep 5
+
+ES_COUNT=$(( NODE_COUNT < 3 ? NODE_COUNT : 3 ))
+
+conduct load conductr-elasticsearch
+conduct run conductr-elasticsearch --scale $ES_COUNT
```

Comment:
Perhaps always scale to 3, or will that be a problem for ES if that scale cannot be attained? @fsat may know... A target scale of 3 will be achieved on a best-effort basis. Therefore if there are less than 3 then all should be well from a ConductR perspective at least.